common/src/main/java/com/wynntils/wynn/model/CompassModel.java

```diff
@@ -57,18 +59,28 @@ public static Optional<WaypointPoi> getCompassWaypoint() {
         return Optional.empty();
     }
 
+    public static Texture getTargetIcon() {
+        return targetIcon;
+    }
+
     public static void setDynamicCompassLocation(Supplier<Location> compassSupplier) {
```

Comment:
Why? Imo, we should still take a texture as argument. Ofc, you can have a method without texture, which defaults to Texture.WAYPOINT. 

We don't need a texture supplier here as even `DynamicIconPoi` has a static icon texture.